modules/nextflow/src/main/groovy/nextflow/NF.groovy

```diff
@@ -32,6 +32,10 @@ class NF {
         return (Session)Global.session
     }
 
+    static String getSyntaxParser() {
```

Comment:
Using `getSyntaxParserVer` would not be a bit more explicit? otherwise it seems it return the parser itself